### PR TITLE
add wide resnet 16/28 variants

### PR DESCRIPTION
**Pull request by brettkoonce**

### CODE REVIEW

File: CIFAR/wide_resnet.swift [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+        featureCounts: (Int, Int), 
+        kernelSize: Int = 3, 
+        widenFactor: Int = 1, 
+        initialStride: (Int, Int) = (2, 2)
+    ) {
+        self.block1 = BatchNormConv2DBlock(
+            filterShape: (kernelSize, kernelSize, featureCounts.0, featureCounts.1 * widenFactor),
+            strides: initialStride)
+        self.block2 = BatchNormConv2DBlock(
+            filterShape: (kernelSize, kernelSize, featureCounts.1 * widenFactor, featureCounts.1 * widenFactor),
```
Comment (rxwei) [inline CIFAR/wide_resnet.swift]: Fit all lines within 100 columns.

Comment (brettkoonce) [reply CIFAR/wide_resnet.swift]: reformatted!

**Comment by rxwei** [comment]
Thank you!

**Comment by brettkoonce** [comment]
@rxwei ugh, forgot i submitted this against stable, want me to submit a pr against master?

**Comment by rxwei** [comment]
Oh, i didn't notice. Could you merge stable to master?

**Pull request merged**